com.regnosys.rosetta/model/Rosetta.xcore

```diff
@@ -287,20 +287,30 @@ enum RosettaSynonymRef {
  * docReferences
  */
 
-class RosettaDocReference extends RosettaRegulatoryBody {
+class RosettaDocReference {
+	contains RegulatoryDocumentReference docReference
+	contains DocumentRationale[] rationales
+	String structuredProvision
 	String provision
-	contains RosettaSegmentRef[] segments
 	boolean reportedField
+	// @Compat. In DRR, there is a segment `table` located after a `rationale`. This should never be the case, but to remain backwards compatible, we need to allow this.
+	contains RosettaSegmentRef[] extraneousSegments
+}
```

Comment:
Cleanup 2: added `rationale`, `rationale_author` and `structured_provision` as actual keywords rather than document segments.